backend/src/zango/apps/auditlogs/registry.py

```diff
@@ -264,7 +257,7 @@ def _get_exclude_models(
         ]
         return exclude_models
 
-    def _register_models(self, models: Iterable[Union[str, Dict[str, Any]]]) -> None:
+    def _register_models(self, models: Iterable[str | dict[str, Any]]) -> None:
```

Comment:
@DevilsAutumn This is also breaking in python 3.9.6
```
    class AuditlogModelRegistry:
  File "/Users/harshshah/Documents/Zelthy_DEV/Zelthy-V2/zel3_main_dev/zelthy3/backend/src/zango/apps/auditlogs/registry.py", line 260, in AuditlogModelRegistry
    def _register_models(self, models: Iterable[str | dict[str, Any]]) -> None:
TypeError: unsupported operand type(s) for |: 'type' and 'types.GenericAlias'
```

Reply:
fixed.